tox.ini

```diff
@@ -16,55 +16,41 @@ envlist =
 # that our tests use.
 setenv = LC_CTYPE = en_US.UTF-8
 # Pass Display down to have it for the tests available
-passenv = DISPLAY TRAVIS TRAVIS_JOB_ID TRAVIS_BRANCH
-whitelist_externals=convert
-# xcffib has to be installed before cairocffi
-deps =
-    pytest >= 6.2.1
-    pytest-cov >= 2.10.1
-    setuptools >= 40.5.0
-    xcffib >= 0.8.1
+passenv = DISPLAY
+# xcffib has to be installed before cairocffi, so we can't use deps anywhere else
+deps = -r requirements/xcffib
 commands =
-    python3 setup.py install
+    pip install -q -r requirements/test -e .
```

Comment:
why -q and -e?

Reply:
-q hides the output, pip output seems unnecessarily loud for our purposes. the `-e .` is instead of calling `python3 setup.py install`

Reply:
you can `pip install .` without `-e`, and I don't think the editable mode is necessary here

Reply:
fair enough